Homework_12/Task_1.py

```diff
@@ -0,0 +1,25 @@
+class Apple:
+    def __init__(self, name, founded_year, center, ceo, products):
```

Comment:
I am not sure that we need the name parameter here. Because we have class Apple. But ok